org.eclipse.xtext.xbase.lib/build.gradle

```diff
@@ -5,3 +5,23 @@ description = 'The runtime library for Xbase languages.'
 dependencies {
 	compile "com.google.guava:guava:$versions.guava"
 }
+
+task copyGwtSources(dependsOn: 'build') {
+	doLast {
+	
+		ant.copy(toDir: '../org.eclipse.xtext.xbase.lib.gwt/build/gwtSources', verbose: true) {
```

Comment:
Gradle has a Copy task from which you can inherit. I think that's better than using the ant bridge.